src/components/CodeOfConduct/CodeOfConduct.tsx

```diff
@@ -22,21 +23,21 @@ const CodeOfConduct = () =>{
             <Title>{t("codeOfConduct:title")}</Title>
 
             <Title style={{width: "100%", marginTop: "10px"}} level={2}>{t("codeOfConduct:introductionSection")}</Title>
-            <p style={{width: "100%"}}>
+            <CodeOfConductParagraph>
```

Comment:
We don't need to be that specific. A general component called `Paragraph` is better.